Classes/EventStore/EventStore.php

```diff
@@ -52,6 +55,23 @@ public function get(EventStreamFilterInterface $filter): EventStream
         return $eventStream;
     }
 
+    /**
+     * Returns the EventAndRawEvent instance that matches the given $eventIdentifier - or throws an EventNotFoundException if no matching event could be found
+     *
+     * @param string $eventIdentifier
+     * @return EventAndRawEvent
+     * @throws EventNotFoundException
+     */
+    public function getOne(string $eventIdentifier): EventAndRawEvent
+    {
+        $rawEvent = $this->storage->loadOne($eventIdentifier);
```

Comment:
What is the actual overhead of using `->load(new EventIdentifierFilter($eventIdentifier))->current();` (with checks and proper error handling) here instead of the custom `loadOne` implementation (now that #167 is merged)? That would also get rid of the ugly code part below, no?